docs/modules/ROOT/pages/index.adoc

```diff
@@ -30,3 +30,7 @@ The table below provides a quick overview of the various AppCat services and the
 
 Get started with xref:object-storage/create.adoc[Create an Object Storage Bucket].
 
+=== Exoscale DBaaS
+
+Redis::
+Get started with xref:exoscale-dbaas/redis/create.adoc[Creating a Redis instance].
```

Comment:
For "Object Storage Buckets" we have a table which describes in which zone a service is available. We need this information here as well... Maybe add a note at after the title saying that these services are only available in Exoscale zones?